# Chapter 0: Introduction

Viruses evolve rapidly, accumulating mutations that escape immunity, improve transmission, and facilitate adaptation to new pathogenic niches. This rapid evolution makes it difficult to develop successful vaccines and therapies. However, by understanding viral evolution, we can anticipate potential changes and develop more effective strategies for prevention and treatment.

High-throughput technologies like deep sequencing and deep mutational scanning are instrumental in this effort, providing detailed insights into the viral genome and proteome. Deep sequencing allows us to identify genetic variations within virus populations, while deep mutational scanning links these variations to their functional consequences. Together, these tools can offer a comprehensive picture of how viruses adapt and resist therapeutic interventions.

The power of deep sequencing lies in its ability to identify variant alleles and quantify their abundance even if they constitute a small fraction of the virus population. By enabling us to map the distribution of genetic variants over time and space, deep sequencing can reveal the intricate details of viral evolution, even at the scale of individual infections. This fine-grained approach has enabled us to understand the implications of viral diversity for immune evasion, drug resistance, and host adaptation.

Although deep sequencing allows us to observe the genetic variation in virus populations within and between hosts, these experiments are often limited to naturally occurring mutations. With deep mutational scanning (DMS), we can extend deep sequencing to explore the functional impact of a massive number of viral mutations in parallel. This technique has made it possible to uncover the impact of amino acid variants on a diverse range of viral phenotypes.

In my graduate work, I built upon these two techniques by developing and employing computational tools to contribute novel insights into the mechanisms that drive virus evolution and adaptation. In this chapter of my dissertation, I’ll provide an overview of what we’ve learned from these techniques, I’ll explain how my work fits into the broader field, and finally, I’ll give a brief overview of the chapters to follow.

## High-throughput experiments in viral evolution

High-throughput experiments provide a systematic view of viral evolution that was previously unachievable. In parallel, advancements in computational biology have allowed us to make sense of the vast datasets produced by these experiments. Together, the synergy between experiment and computation has led to significant breakthroughs in understanding the evolutionary pressures that shape viral fitness. In my dissertation, I will focus on two high-throughput techniques; deep sequencing and deep mutational scanning. In the following sections, I will describe each of these techniques in detail alongside the contributions that they've made to our understanding of viruses. I will also highlight the challenges that complicate their analysis and interpretation.

### Deep sequencing: studying viral evolution at multiple scales

The development of genomic sequencing is tightly linked with the study of viruses. In fact, the very first genome sequenced was that of a virus, the bacteriophage ΦX174. Over four decades later, access to affordable deep sequencing has revolutionized our approach to studying viral evolution by allowing us to investigate genomic variants with unprecedented resolution. This resolution is particularly important for studying RNA viruses, which, due to their high mutation rates, exist within a host as an ensemble of diverse but related viral particles.

This diversity is an import feature of viral infections that couldn't be adequately explored with previous sequencing methods. For example, methods like Sanger sequencing only capture a consensus sequence that doesn't necessarily reflect the underlying population of viral variants. Although coupling Sanger sequencing with the isolation of individual viral clones can provide a more detailed picture of a viral population, this approach is cumbersome and still only provides limited resolution. In contrast, deep sequencing can accurately identify viral variants that make up even a miniscule fraction of a sample, opening up the door to study the dynamics of viral populations at the scale of individual infections.

The research presented in the upcoming chapters primarily focuses on the within-host dynamics of Measles and SARS-CoV-2. Both are RNA viruses that typically cause acute respiratory illness. To keep the scope of this section as relevant as possible, I'll focus on the within-host evolution of these and related acute RNA viruses.

#### Exploring viral dynamics within individual hosts

During a viral infection, error-prone replication leads to an accumulation of genetic diversity in a host. Evolutionary forces like selection and genetic drift can act on this viral diversity leading to drug resistance, immune evasion, and host adaptation. However, the extent to which selective pressure drives viral evolution within an individual host depends on the circumstances of the infection.

In any given host, there are multiple factors that could drive the evolution of a viral population. For example, the virus might be confronted with an immune response or an antiviral therapy. Even in the absence of therapeutic treatment, the virus has to contend with a complicated spatial structure of host cells, many of which might not be permissible to infection. These factors all provide strong pressures that can drive viral evolution. In other words, mutations that give a viral variant and advantage by blunting the host immune system, resisting therapeutic treatment, or increasing the efficiency of infecting host cells will have a selective advantage.

Given the many selective pressures that could act on the virus population in a host, a reasonable expectation would be that signatures of selection would be common in most infections. However, in most studies of *acute* infections – like those caused by SARS-CoV-2 and Influenza – there is very limited genetic diversity at an observable frequency. As noted by Xue et al., this observation can be explained, in part, by a simple birth model. Given the short length of acute infection, there isn't enough time for the viral population to accumulate mutations at a higher frequency than the error rate of most deep-sequencing techniques. This explains why most studies of SARS-CoV-2 and Influenza observed between 3 and 10 unique variants, most at fairly low frequency. Given this lack of diversity and the delay between the peak of viral infection and the adaptive immune response, it's unsurprising that it's very rare to observe signatures of selection in acute viral infections.

However, viruses – even those that typically cause short-lived infections – can persist in the host due to factors like a weakened immune system. In contrast to acute infections, these *chronic* infections provide a virus amble opportunity to diversify in the presence of selective pressures. By studying chronic infections, we can observe how viruses evolve to escape host immunity, develop drug resistance, and adapt to the host environment. For example, the mutations observed in studies of chronic influenza infections have been fairly predictive of antigenic drift in the global population. In HIV, the virus can occupy compartments within the host that aren't permeable to antiviral drugs, leading to the development of resistance. Finally, for SARS-CoV-2, chronic infections are postulated to be the origin of variants that evade population immunity and transmit more easily. While the exact role that chronic infections play on global evolution have yet to be determined, it is appreciated that these infections provide a rich opportunity to understand viral evolution in a natural experiment.

Despite the utility of studying within-host evolution and chronic infection, there remain some significant limitations with these types of experiments. To get an accurate measurement of viral diversity, one must account for errors due to sequencing and PCR amplification. It has been shown repeatedly that samples with low viral load, or low 'effective depth,' can lead to significantly skewed estimates of the frequency of variant alleles. When possible, using an experimental design that controls for mutations is preferred. When that's not possible, sequencing samples in replicates from separate RT-PCR reactions, or avoiding targeted amplification altogether, is preferred. Finally, taking a careful computation approach to remove false positives as stringently as possible is necessary.

Additionally, most deep sequencing studies use short reads between 100 and 300 base pairs. These short reads are incapable of linking mutations to uncover the true genotype of the viral molecule they originated from. It has been shown that long-read sequencing in combination with short read approaches can phase viral haplotypes, however, these methods are still fairly uncommon. Additionally, there have been many computation approaches aimed at constructing viral haplotypes from short reads. However, these approaches are still limited by low sensitivity.

In the work presented in chapter 1, I'll discuss

#### Uncovering the role of transmission on global viral evolution

RNA viruses like SARS-CoV-2 and Influenza rapidly evolve by fixing mutations that allow them to evade population-level immune pressure. At some point, these adaptive mutations originated as *de novo* mutations within individual infections as the result of error-prone replication. Therefore, the viral diversity that arises within infected hosts provides the substrate for population-level evolution.

Transmission links the evolutionary dynamics within a host to evolution at the population level. Upon transmission to a new host, the genetic diversity generated within a host is subject to a bottleneck. Specifically, I will refer to the *transmission bottleneck* as the effective size of the virus population from a donor host that establishes a new infection in a recipient host. This bottleneck’s size profoundly impacts selection efficiency and the rate of adaptation between scales. If the bottleneck is wide, the genetic diversity that arose in the donor host will be preserved in the recipient, maintaining the selective pressures that acted on that population. However, if the bottleneck is narrow, the effective population size between hosts is small, and genetic drift will dominate as the primary evolutionary force.

Deep sequencing has made it possible to quantitatively measure the size of the transmission bottleneck. It's possible to sequence a donor host and recipient host and compare the viral population. Using this information, there are several methods for calculating the transmission bottleneck. Typically, these methods treat transmission as a binomial sampling process. 

Several studies have used these methods to calculate the transmission bottleneck. Surprisingly, in almost all cases the bottleneck is very narrow. In SARS-CoV-2 the bottleneck is between 1 and 3 founding variants. In influenza it's about the same. 

However, there are significant limitations to these methods. For instance, recurrent sequencing error can lead to significant overestimation of the bottleneck size. Although heuristic thresholds can ameliorate these problems, overly stringent filter can lead to underestimation of the true bottleneck size. Furthermore, in acute infections, there is often limited detectable genetic diversity. This significantly reduces the power to calculate the bottleneck size. There are studies that use a barcoding approach, but these are necessarily limited to animal models. Finally, most studies of the transmission bottleneck don't use full haplotypes to calculate the bottleneck and therefore can be confounded by linkage between alleles. Although there are methods that take this into account, for reasons stated in the previous section, it's difficult to accurately construct viral haplotypes.

Despite the progress in calculating transmission bottlenecks, there are still some significant gaps in our knowledge. For example, most studies of the transmission bottleneck of viruses are in household or hospital settings. These types of transmission scenarios have the advantage of straightforward contact tracing. However, they're aren't necessarily reflective of the transmission events that drive global spread of viruses like SARS-CoV-2.

Although better experimental approaches can help accurately calculate the true bottleneck size, the dynamics of viral variation can still provide clear indications about the relative size of the transmission bottleneck.

### Deep mutational scanning: systematically profiling the impact of viral mutations

#### Overcoming hurdles to decipher mutational data

## Layout of Dissertation
