# Chapter 0: Introduction

Viruses evolve rapidly, accumulating mutations that escape immunity, improve transmission, and enable adaptation. This rapid evolution makes it difficult to develop successful vaccines and therapies. However, by studying viral evolution, we can attempt to anticipate adaptive mutations and develop more effective strategies for prevention and treatment.

High-throughput technologies like deep sequencing and deep mutational scanning are a crucial part of this effort, providing detailed insights into the viral genome and proteome. Deep sequencing allows us to identify genetic variation within viral populations, while deep mutational scanning links viral variants to their functional consequences. Together, these tools offer a comprehensive picture of how viruses adapt and resist therapeutic interventions. *In my graduate work, I built upon these two techniques by developing and employing computational tools to contribute novel insights into the mechanisms that govern virus evolution.*

## High-throughput experiments in viral evolution

High-throughput experiments provide a systematic view of viral evolution that was previously unachievable. In parallel, computational biology allows us to make sense of the vast datasets produced by these experiments. This synergy between experiment and computation has led to significant breakthroughs in our understanding of the evolutionary pressures that shape viral fitness. In my dissertation, I will focus on two high-throughput techniques; *deep sequencing* and *deep mutational scanning*.

The power of deep sequencing lies in its ability to identify viral variants and quantify their abundance even if they constitute a small fraction of the viral population. By enabling us to map the distribution of genetic variants over time and space, sequencing can reveal the dynamics of viral evolution at the scale of individual infections. However, these experiments are often limited to naturally occurring mutations. With *deep mutational scanning (DMS)*, we can extend deep sequencing to explore the functional impact of a massive number of viral mutations in parallel.

In this chapter, I’ll provide an overview of what we’ve learned from these techniques and explain how my graduate work fits into the broader field.

### Deep sequencing: studying viral evolution at multiple scales

The development of genomic sequencing is tightly linked with the study of viruses. In fact, the very first genome sequenced was that of a virus, the bacteriophage ΦX174 [@sangerNucleotideSequenceBacteriophage1977]. Over four decades later, access to affordable deep sequencing has revolutionized the study of viral evolution by enabling us to capture high resolution snapshots of the viral population. This resolution is particularly important for studying RNA viruses, which, due to their high mutation rates, exist within a host as diverse ensemble of viral genomes [@domingoNucleotideSequenceHeterogeneity1978].

This diversity is an important feature of viral infections that couldn't be easily explored with previous sequencing methods. For example, Sanger sequencing only captures a consensus sequence that doesn't necessarily reflect the underlying population of viral variants. Although Sanger sequencing individual viral clones isolated from an infection can provide a more detailed picture of a viral population, this approach is cumbersome and still provides only limited resolution. In contrast, deep sequencing can accurately identify viral variants that make up even a miniscule fraction of a sample, opening up the door to study the dynamics of viral populations at the scale of individual infections [@beerenwinkelUltradeepSequencingAnalysis2011].

The research presented in the upcoming chapters primarily focuses on the within-host dynamics of Measles and SARS-CoV-2. Both are RNA viruses that cause acute respiratory illness. To keep the scope of this section as relevant as possible, I'll focus on the within-host evolution of these and related RNA viruses.

#### Exploring viral dynamics within individual hosts

During a viral infection, error-prone replication leads to an accumulation of genetic diversity [@hollandRapidEvolutionRNA1982]. Evolutionary forces like selection and genetic drift act on this viral diversity to shape viral evolution in the host. For instance, the viral population might be confronted with selective pressures like an immune response or an antiviral therapy. Even in the absence of therapeutic treatment, the viral population has to contend with a complicated spatial structure of host cells, many of which cannot be infected. However, the breath and influence of selective forces on the viral population depends on the circumstances of the infection.

Given the numerous selective pressures acting in a host, a reasonable expectation is that we will observe signatures of selection in most infections. However, in studies of *acute* infections like those caused by SARS-CoV-2 and Influenza, selection is rarely observed []. The apparent dominance of genetic drift in acute infections is likely the result of several factors. For one, these infections tend to accumulate limited genetic diversity. In clinical samples isolated from patients with SARS-CoV-2 and Influenza, it's typical to identify as few as 3 to 10 unique viral variants, most of which make up a small fraction of the total viral population []. Despite the high error rate of RNA viruses, the short length of acute infections does not provide enough time for the viral population to accumulate mutations that can be measured by deep sequencing []. Another factor that has been hypothesized to contribute to the lack of apparent selection in acute infections is the lag between the timing of peak viral load and the peak of adaptive immunity or antiviral treatment []. Finally, recent evidence suggests that the viral population faces significant population bottlenecks within hosts []. All of these factors play a role in the stochastic nature of viral evolution during acute infections.

Occasionally, viruses that cause short-lived infections can persist in the host due to factors like a weakened immune system. In contrast to acute infections, these *chronic* infections provide a viral population ample opportunity to diversify in the presence of selective pressures like host immunity and therapeutic treatments. Chronic infections have acted as a natural experiment to study how viruses evolve to escape host immunity, develop drug resistance, and adapt to the host environment []. Additionally, samples can often be collected longitudinally from chronic infections, allowing for a detailed analysis of viral evolution over time. For instance, longitudinal samples collected from a study of chronic influenza infections contained viral variants with antigenic mutations that parallelled antigenic drift in the global population []. Studies of chronic SARS-CoV-2 infections have demonstrated similar patterns []. In fact, chronic infections are postulated to be the origin of SARS-CoV-2 'variants of concern' that evade population-level immunity and transmit more easily []. While the exact role that chronic infections play in global evolution have yet to be fully determined, it is appreciated that these infections provide a valuable insight into the forces shaping viral evolution.

Despite the utility of using deep sequencing to study viral evolution within hosts, there remain some significant limitations with these kinds of experiments. To get an accurate measurement of viral diversity, one must account for errors from to sequencing and PCR amplification []. It has been shown repeatedly that samples with low viral load, or low 'effective depth,' can lead to significantly skewed estimates of the frequency of variant alleles []. When possible, using an experimental design that controls for mutations is preferred []. When that's not possible, sequencing samples in replicates from separate RT-PCR reactions and avoiding targeted amplification can help mitigate these issues []. Finally, no matter what the experimental design is, it's necessary to take a careful computational approach to remove false positives [].

Another issue facing many deep sequencing studies of viral infections is the lack of information about the linkage between mutations. The most common sequencing approaches produce short-reads ranging between 100 and 300 nucleotides long []. Using these short reads, it's very difficult to phase mutations and determine if mutations arose on the same viral genomes. Although it is possible to augment short reads with long-read approaches like PacBio or Oxford Nanopore sequencing, the high cost and high error rates are a limiting factor []. To combat this, there have been several computational approaches aimed at determining the linkage of single mutations from short reads []. However, these approaches are limited by low sensitivity and high rates of false-positives []. It's not surprising then that most studies of viral populations within hosts consider all mutations as independent. This is a significant blind-spot that can make it difficult to uncover phenomenons like clonal dynamics and phylogenetic relationships.  

In the work presented in **Chapter 1**, I'll discuss how we used deep sequencing data from a chronic Measles infection to characterize the adaptation of Measles to an atypical tissue niche, the brain. Metagenomic sequencing was performed on samples collected from spatially-distributed regions of a human brain. Although we performed short read Illumina sequencing, we developed a computation approach to take advantage of spatial sampling to resolve viral haplotypes and determine their phylogenetic relationship. Oxford Nanopore sequencing was then used to confirm the identity of the major haplotypes. Our haplotype-resolved approach made it possible to shed light on clonal dynamics that might have enable Measles to spread in the brain.

#### Uncovering the role of transmission on global viral evolution

Transmission links the evolutionary dynamics within a host to evolution in the global population. Viruses like SARS-CoV-2 and Influenza evolve by fixing mutations that allow them to evade population-level selective pressures like immunity and antiviral treatments. At some point, these adaptive mutations originated as *de novo* mutations within individual infections. Upon transmission to a new host, this genetic diversity is subjected to a bottleneck. Specifically, I will refer to the *transmission bottleneck* as the size of the founding population of virions that establishes lineages leading to a new infection in a recipient host []. This bottleneck’s size profoundly impacts the efficiency of selection. If the bottleneck is "loose", the genetic diversity that arose in the first host will be preserved in the infected host, maintaining the selective pressures that acted in the previous infection. However, if the bottleneck is "narrow", the effective population size between hosts is small, and genetic drift will dominate as the primary evolutionary force [].

Deep sequencing has made it possible to experimentally assess the size of the transmission bottleneck. To illustrate this, imagine that we have identified a contact pair where one individual, the *donor*, has infected another individual, the *recipient*. By sequencing the viral population of this *donor-recipient* pair, we can identify the variant alleles in each population of viruses. If we assume that transmission represents a sampling process from the viral population in the *donor*, we can determine the size of that sampling event by comparing the variants present in the *donor* to the variants present in the *recipient*. If the majority of viral variants are observed in both the *donor* and *recipient*, we can assume that the bottleneck is quite "loose", since a large sample, or founding population, would be needed to preserve the population structure of the donor. In contrast, if most variants are not shared, or if some variants randomly become fixed in the *recipient*, this indicates a small sample, or a "narrow" bottleneck.

Several methods have been developed to quantitatively measure the size of the transmission bottleneck. One of the most widely used approaches, developed by Sobel-Leonard et. al., uses a `beta-binomial` model that treats transmission as a binomial sampling process while accounting for stochastic variation in the viral population between the time of infection and the time of sequencing []. Using this approach and others, the transmission bottleneck has been estimated for a wide array of viruses, including Influenza, SARS-CoV-2, HIV, and various plant viruses []. In nearly all cases, the bottleneck is narrow, restricting the founding population to anywhere between 1 to 15 successfully established virions []. However, existing methods only consider the variants that are present in the *donor* while ignoring the *de novo* genetic variants that arise in the *recipient*. By restricting themselves to the genetic variation generated in the donor, these methods are very susceptible to stochastic changes in the *donor* population that would lead to an underestimation of the true bottleneck size. Recently, the developers of the `beta-binomial` method addressed this by creating an approach to estimate the transmission bottleneck by modeling the accumulation of clonal variants in the *recipient* as a multi-state branching process []. Despite accounting for possible underestimation of the bottleneck size, Shi et. al., confirmed that the transmission bottleneck for both SARS-CoV-2 and Influenza is narrow [].

Although there have been many advances made in the methods used to calculate the transmission bottleneck, there remain some significant limitations to these studies. For instance, recurrent sequencing error can lead to significant overestimations of the bottleneck size []. As a result, stringent heuristic filters are often applied to the identified variants to remove false-positives. Although these filters are successful at removing false-positives, overly stringent filters could lead to *underestimations* of the true bottleneck size. Furthermore, as mentioned in the previous section, there is often limited and stochastic genetic diversity in acute infections that reduces the power to calculate the bottleneck size []. Some groups have used nucleotide barcodes to add diversity and improve the accuracy of transmission bottleneck estimations []. However, these experiments are necessarily limited to animal models of transmission which don't necessarily recapitulate the dynamics of natural infections in humans. Finally, most approaches used to estimate the transmission bottleneck consider all mutations to be independent, potentially causing their estimates to be confounded by the linkage between variant alleles. As a result, a method has been developed to augment the `beta-binomial` approach with information about the linkage between mutations []. However, for reasons stated in the previous section, it's often difficult to accurately phase viral haplotypes from short-reads. In spite of these limitations, there is strong evidence that transmission imposes a narrow bottleneck.

However, there remain many gaps in our understanding of transmission bottlenecks. One of the most glaring is how the circumstances of transmission impact the size of the bottleneck. The majority of bottleneck estimates are from cohorts in household or hospital settings. These scenarios have the advantage of straightforward contact tracing. However, they aren't necessarily reflective of the transmission events that drive the global spread of viruses. For instance, super-spreading events may have played an outsized role in the global spread of SARS-CoV-2 []. We hypothesized that the same factors that cause super-spreading events – close quarters, prolonged contact, and higher viral loads – could lead to a wide transmission bottleneck. In **Chapter 2**, we tested this by investigating how viral diversity spread among crew members from a fishing that experienced an outbreak of SARS-CoV-2 with a high attack rate. Despite the scenario being highly conducive to transmission, we identified all of the hallmarks of a narrow bottleneck.

### Deep mutational scanning: systematically profiling the impact of viral mutations

Deep mutational scanning (DMS) is an experimental approach used to determine the phenotype of a massive number protein variants simultaneously []. In contrast with earlier techniques, DMS is not limited to analyzing a set of mutations selected using *a priori* knowledge about their possible significance. Instead, DMS provides an *unbiased* approach to profile the effects of every possible mutation in a protein sequence. The unbiased nature of DMS allows for the systematic investigation of phenotypes ranging from biochemical properties to evolutionary constraints []. In this section, I'll focus on how DMS has been used to study viruses.

#### An overview of the design of deep mutational scans

The design of a mutational scan involves three steps; creating a library of protein variants, imposing a selective pressure, and comparing the library's composition before and after selection. The exact details of each step depends on the context of the experiment. For example, multiple approaches are available to generate the mutant library, including random mutagenesis, codon-directed mutagenesis, and mutant oligonucleotide synthesis []. Additionally, there infinite types of selection to apply to the mutant library, each of which dictates the genotype-phenotype relationship. Finally, the method for calculating the impact of each genotype on the measured phenotype depends on the library design, type of selection, and sequencing approach.

There have been many methods developed to perform deep mutational scans on viral proteins. One of the most straightforward approaches involves the use of reverse genetics systems to create a library of viruses with the mutant protein of interest. This library can then be used to infect cells under a wide array of conditions. Reverse genetics systems have been used to perform DMS on a variety of viral accessory, structural, and receptor proteins []. However, making mutations to 'live' viral genomes poses risks of the creation and accidental release of more virulent strains, leading to serious biosafety concerns. In contrast, surface display offers a safe, alternative, approach to perform DMS on viral proteins. In this case, DMS is achieved by displaying a library of mutant peptides on the surface of a cell, or virus, while maintaining the link between genotype and phenotype. There are many types of surface display and each has it's advantages and disadvantages. Yeast can display large libraries of mutant peptides, but the maximum size of the displayed peptide is smaller than most viral proteins []. Bacteriophage are another popular choice due to superior library size and ability to map mutations on an entire protein []. However, the large library size comes at the cost of using small linear peptides that lack the structural context of the folded protein. Finally, mammalian cells can display entire viral proteins while preserving their structure and relevant post-translational modifications []. However, as with all surface display methods, mammalian cell display is limited to measuring biophysical properties like binding and stability rather than infection. Recently, an approach has been developed to perform deep mutational scanning on viral glycoproteins using lentiviral pseudotyping []. Unlike surface display methods, this pseudotyping approach can capture the effect of mutants on *infection*. However, due to the restricted replication of pseudotyped lentiviruses, this system lacks the biosafety concerns of using a 'live virus'.

#### Deep mutational scanning has contributed significantly to our understanding of viruses

Each of the approaches mentioned in the previous section have contributed significantly to our understanding of viruses and viral evolution. For example, reverse genetics systems have been used to systematically map the effect of mutations to viral proteins on fitness for diverse viruses like HIV, Influenza, SARS-CoV-2, and Zika, revealing the mutational constraints on viral proteins, many of which are the target of therapeutic interventions []. Additionally, surface display methods have been used to measure the effect of mutations in viral glycoproteins on host receptor affinity, antibody binding, and stability []. More recently, pseudotyped lentivirus, along with a biophysical model of antibody binding, has been used to map the antibody binding footprints of polyclonal sera for viruses like HIV and SARS-CoV-2 [].

The information generated by these experiments can shed light on the dynamics of viral evolution. For example, yeast display of the SARS-CoV-2 receptor binding domain was used to prospectively identify mutations that could escape therapeutic antibodies while maintaining receptor binding []. The mutations identified in this study were subsequently found in patients treated with these therapies and in the global population []. DMS has also been used to identify mutations that shift host specificity and therefore could increase the risk of zoonosis. For example, DMS of a pandemic influenza virus identified residues that shifted host specificity from pig to human []. Furthermore, DMS can be used to identify epistasis in viral protein evolution. For example, by performing DMS on the SARS-CoV-2 receptor binding domain in multiple strains, combinations of mutations have been identified that have non-additive effects on receptor binding [].

#### Overcoming hurdles in the analysis of mutational data

Although DMS has proven incredibly useful for studying viruses and viral evolution, there are still some significant limitations. For instance, the number of mutant sequences in the library pales in comparison to the sequence space of available to an evolving protein. Moreover, the effects of mutations often combine in non-additive ways due to epistasis. This poses an issue, because epistasis plays a significant role in viral evolution. For example, it's been shown that epistasis between mutations in SARS-CoV-2 Spike have allowed it to evade immunity while retaining high binding affinity to the host receptor []. There have been studies that compare DMS from multiple genetic backgrounds to identify epistasis [].

Another limitation of DMS stems from the complexity of the analysis. Given the vast number of experimental designs and questions, it's challenging to develop a consistent approach for analyzing mutational scanning data. As a result, the analysis can impose a significant bottleneck for those lacking computational expertise. There have been attempts to develop user friendly softwares to address these hurdles. For example, programs like `dms-tools` and `Enrich` provide consistent statistical frameworks for inferring the impact of mutations from DMS data []. Additionally, libraries like `mutagenesis_visualization` are aimed at at simplifying the visualization steps of the analysis []. However, until recently, there wasn't a straightforward approach for analyzing mutation-based data in the context of a 3D protein structure. Most researches were forced to take an `ad hoc` approach involving multiple steps and softwares. To address this, our lab developed `dms-view`, an in interactive tool that integrates key visualizations with an interactive 3D protein structure, massively simplifying this aspect of analysis [].

In **chapter 3**, I'll discuss how I designed a new web-based tool – `dms-viz` – for visualizing mutational data in the context of an interactive 3D protein model. I built `dms-viz` to be customizable and comprehensive in order to handle a wide diversity of experimental designs and questions. Additionally, I created a command line tool called `configure-dms-viz` to make it straightforward to automate the process of formatting mutation-based data for visualization. Because `dms-viz` is capable of handling data from diverse experimental designs, it can be used to visualize a wide range of mutation-based datasets with ease.
