# Chapter 0: Introduction

Viruses evolve rapidly, accumulating mutations that escape immunity, improve transmission, and facilitate adaptation to new pathogenic niches. This rapid evolution makes it difficult to develop successful vaccines and therapies. However, by understanding viral evolution, we can anticipate potential changes and develop more effective strategies for prevention and treatment.

High-throughput technologies like deep sequencing and deep mutational scanning are instrumental in this effort, providing detailed insights into the viral genome and proteome. Deep sequencing allows us to identify genetic variation within viral populations, while deep mutational scanning links viral variants to their functional consequences. Together, these tools can offer a comprehensive picture of how viruses adapt and resist therapeutic interventions.

The power of deep sequencing lies in its ability to identify variant alleles and quantify their abundance even if they constitute a small fraction of the virus population. By enabling us to map the distribution of genetic variants over time and space, deep sequencing can reveal the intricate details of viral evolution, even at the scale of individual infections. *This fine-grained approach has enabled us to understand the implications of viral diversity for immune evasion, drug resistance, and host adaptation.*

Although deep sequencing allows us to observe the genetic variation in virus populations within and between hosts, these experiments are often limited to naturally occurring mutations. With deep mutational scanning (DMS), we can extend deep sequencing to explore the functional impact of a massive number of viral mutations in parallel. *This technique has made it possible to uncover the impact of amino acid variants on a diverse range of viral phenotypes.*

In my graduate work, I built upon these two techniques by developing and employing computational tools to contribute novel insights into the mechanisms that govern virus evolution. *In this chapter of my dissertation, I’ll provide an overview of what we’ve learned from these techniques, I’ll explain how my work fits into the broader field, and finally, I’ll give a brief overview of the chapters to follow.*

## High-throughput experiments in viral evolution

High-throughput experiments provide a systematic view of viral evolution that was previously unachievable. In parallel, advancements in computational biology have allowed us to make sense of the vast datasets produced by these experiments. Together, the synergy between experiment and computation has led to significant breakthroughs in understanding the evolutionary pressures that shape viral fitness. In my dissertation, I will focus on two high-throughput techniques; deep sequencing and deep mutational scanning. In the following sections, I will describe the contributions that these techniques have made to our understanding of viruses. I will also highlight the challenges that complicate their analysis and interpretation.

### Deep sequencing: studying viral evolution at multiple scales

The development of genomic sequencing is tightly linked with the study of viruses. In fact, the very first genome sequenced was that of a virus, the bacteriophage ΦX174. Over four decades later, access to affordable deep sequencing has revolutionized our approach to studying viral evolution by allowing us to investigate genomic variants with high resolution. This resolution is particularly important for studying RNA viruses, which, due to their high mutation rates, exist within a host as an ensemble of diverse but related viral particles.

This diversity is an important feature of viral infections that couldn't be easily explored with previous sequencing methods. For example, methods like Sanger sequencing only capture a consensus sequence that doesn't necessarily reflect the underlying population of viral variants. Although coupling Sanger sequencing with the isolation of individual viral clones can provide a more detailed picture of a viral population, this approach is cumbersome and still only provides limited resolution. In contrast, deep sequencing can accurately identify viral variants that make up even a miniscule fraction of a sample, opening up the door to study the dynamics of viral populations at the scale of individual infections.

The research presented in the upcoming chapters primarily focuses on the within-host dynamics of Measles and SARS-CoV-2. Both are RNA viruses that typically cause acute respiratory illness. To keep the scope of this section as relevant as possible, I'll focus on the within-host evolution of these and related RNA viruses.

#### Exploring viral dynamics within individual hosts

During a viral infection, error-prone replication leads to an accumulation of genetic diversity in a host. Evolutionary forces like selection and genetic drift can act on this viral diversity. In any given host, there are multiple factors that could drive the evolution of a viral population. For example, the virus might be confronted with an immune response or an antiviral therapy. Even in the absence of therapeutic treatment, the virus has to contend with a complicated spatial structure of host cells, many of which are not permissible to infection. These factors all provide strong pressures that can drive viral evolution. However, the extent to which selective pressure drives viral evolution within an individual host depends on the circumstances of the infection.

Given the many selective pressures acting on the virus population in a host, a reasonable expectation is that we should observe patterns of selection in most infections. However, in studies of *acute* infections like those caused by SARS-CoV-2, Influenza, and Measles, selection is rarely observed. The apparent dominance of genetic drift in acute infections is likely the result of several factors. For one, these infections tend to accumulate limited genetic diversity. In clinical samples isolated from patients with SARS-CoV-2 and Influenza, it's typical to observe as few as 3 to 10 unique viral variants, most of which make up a small fraction of the total viral population. Despite the high error rate of RNA viruses, the short length of acute infections does not provide enough time for the viral population to accumulate mutations that can be observed by deep sequencing. Another factor that has been hypothesized to contribute to the lack of apparent selection in acute infections is the lag between the timing of peak viral load and the peak of adaptive immunity or antiviral treatment. Finally, it's been observed that the viral population faces significant population bottlenecks within hosts. All of these factors play a role in the stochastic nature of viral evolution during acute infections.

However, viruses that usually cause short-lived infections can persist in the host due to factors like a weakened immune system. In contrast to acute infections, these *chronic* infections provide a virus ample opportunity to diversify, and to do so in the presence of selective pressures like host immunity and antiviral treatment. By studying chronic infections, we can observe how viruses evolve to escape host immunity, develop drug resistance, and adapt to the host environment. Additionally, these cases sometimes offer the opportunity for longitudinal sampling. For example, the mutations observed in studies of chronic influenza infections have been fairly predictive of antigenic drift in the global population. In HIV, the virus can occupy compartments within the host that aren't permeable to antiviral drugs, leading to the development of resistance. Finally, for SARS-CoV-2, chronic infections are postulated to be the origin of variants that evade population-level immunity and transmit more easily. While the exact role that chronic infections play in global evolution have yet to be fully determined, it is appreciated that these infections provide a valuable natural experiment to study viral evolution.

Despite the utility of studying within-host evolution in chronic infections, there remain some significant limitations with these types of experiments. To get an accurate measurement of viral diversity, one must account for errors due to sequencing and PCR amplification. It has been shown repeatedly that samples with low viral load, or low 'effective depth,' can lead to significantly skewed estimates of the frequency of variant alleles. When possible, using an experimental design that controls for mutations is preferred. When that's not possible, sequencing samples in replicates from separate RT-PCR reactions and avoiding targeted amplification can help mitigate these issues. Finally, no matter what the experimental design is, it's necessary to take a careful computational approach to remove false positives.

Another issue facing most deep-sequencing studies of viral infections is the lack of information about the linkage between mutations. Most studies involve deep-sequencing approaches that produce short reads, between 100 and 300 nucleotides long. Using these short reads, it's very difficult to phase mutations and reconstruct the original molecules that mutations arose on. Although it is possible to augment short reads with long-read PacBio or Oxford Nanopore sequencing, higher cost and higher error rates respectively are often a limiting factor. To combat this, there have been many computation approaches aimed at constructing viral haplotypes from short reads. However, these approaches are significantly limited by low sensitivity and high rate of false-positives. It's not surprising then that most studies of viral populations within hosts consider all mutations as independent. This is a significant blind-spot that can make it difficult to uncover things like clonal dynamics.  

In the work presented in **Chapter 1**, I'll discuss how we took advantage of samples from a chronic infections within an individual host to characterize the selective pressures that drove host adaptation of Measles to an atypical pathogenic niche; the brain. Metagenomic sequencing was performed on samples collected from spatially distributed regions of a human brain from an individual who unfortunately succumbed to Subacute Sclerosing Panencephalitis. Although our sequencing method produced short-reads, we developed a computation approach to take advantage of spatial sampling to resolve viral haplotypes and determine their phylogenetic relationship. Long-read Oxford Nanopore sequencing was used to confirm the identity of major haplotypes. Our haplotype-resolved approach made it possible to use our observations on viral evolution to shed light on unknown aspects of Measles pathology.

#### Uncovering the role of transmission on global viral evolution

Transmission links the evolutionary dynamics within a host to evolution in the global population. Viruses like SARS-CoV-2 and Influenza rapidly evolve by fixing mutations that allow them to evade population-level selective pressures like immunity and antiviral treatments. At some point, these adaptive mutations originated as *de novo* mutations within individual infections. Upon transmission to a new host, this genetic diversity is subjected to a bottleneck. Specifically, I will refer to the *transmission bottleneck* as the size of the founding population of virions that establishes a new infection in a recipient host. This bottleneck’s size profoundly impacts the efficiency of selection. If the bottleneck is "loose", the genetic diversity that arose in the first host will be preserved in the infected host, maintaining the selective pressures that acted in the previous infection. However, if the bottleneck is "narrow", the effective population size between hosts is small, and genetic drift will dominate as the primary evolutionary force.

Deep sequencing has made it possible to experimentally assess the size of the transmission bottleneck. To illustrate this, imagine that we have identified a contact pair where one individual, the *donor*, has infected another individual, the *recipient*. By sequencing the viral population of this *donor-recipient* pair, we can identify the variant alleles in each population of viruses. If we assume that transmission represents a sampling process from the viral population in the *donor*, we can determine the size of that sampling event by comparing the variants present in the *donor* to the variants present in the *recipient*. If the majority of viral variants are observed in both the *donor* and *recipient*, we can assume that the bottleneck is quite "loose", since a large sample, or founding population, would be needed to preserve the population structure of the donor. In contrast, if most variants are not shared, or if some variants randomly become fixed in the *recipient*, this indicates a small sample, or a "narrow" bottleneck.

Several methods have been developed to quantitatively measure the size of the transmission bottleneck by modeling it as a binomial sampling process. One of the most widely used approaches, developed by Sobel-Leonard et. al., uses a `beta-binomial` model to also account for the stochastic process that alter the frequency of variants between the time of infection and the time of sampling in the *recipient*. Using this approach and others, the transmission bottleneck has been estimated for a wide array of viruses, including Influenza, SARS-CoV-2, HIV, and various plant viruses. In all cases, the bottleneck appears to be quite narrow, restricting the founding population to anywhere between 1 to 15 virions that successfully establish a lineage in the *recipient*. Recently, the same group that developed the `beta-binomial` approach developed a new approach that accounts for the fact that it's unlikely for the *donor* to be sampled at the time of infection. They hypothesize that the stochastic nature of the viral population in the donor host could cause changes in the donor population from the time of sampling. Since the beta-binomial approach only considers mutations that are present in the donor, this could be leading to an underestimate of the true bottleneck. Instead of only considering mutation in the donor, they focus their approach on the clonal variants that arise in the recipient. By modeling the accumulation of clonal variants as a multi-state branching process, they can more accurately calculate the bottleneck. Despite accounting for possible underestimation of the bottleneck size, Shi et. al., confirmed that the transmission bottleneck for both SARS-CoV-2 and Influenza is narrow.

Although there have been advances made in the computational methods used to calculate the transmission bottleneck size, there remain some significant limitations to these studies. For instance, recurrent sequencing error can lead to significant overestimations of the bottleneck size. As a result, stringent heuristic filters are often applied to the identified variants to remove false-positives. Although these filters are successful at removing false-positives, overly stringent filter could lead to *underestimations* of the true bottleneck size. Furthermore, as mentioned in the previous section, there is often limited and stochastic genetic diversity in acute infections that reduces the power to calculate the bottleneck size. Some groups have used nucleotide barcodes to add diversity and improve the accuracy of transmission bottleneck estimations. However, these experiments are necessarily limited to animal models of transmission which don't necessarily recapitulate the dynamics of natural infections in humans. Finally, most approaches used to estimate the transmission bottleneck consider all mutations to be independent, potentially causing their estimates to be confounded by the linkage between variant alleles. As a result, a method has been developed to augment the beta-binomial approach with information about the linkage between mutations. However, for reasons stated in the previous section, it's often difficult to accurately phase viral haplotypes from short-reads. In spite of these limitations, there is strong evidence that transmission imposes a narrow bottleneck.

However, there remain many gaps in our understanding of transmission bottlenecks. One of the most glaring is how the circumstances of transmission impact the stringency of the bottleneck. The majority of bottleneck estimates are from cohorts in household or hospital settings. These scenarios have the advantage of straightforward contact tracing. However, they aren't necessarily reflective of the transmission events that drive the global spread of viruses. For instance, super-spreading events may have played an outsized role in the global spread of SARS-CoV-2. We hypothesized that the same factors that cause super-spreading events – close quarters, prolonged contact, and higher viral loads – could lead to a wider bottleneck. In **Chapter 2**, we tested this by investigating how viral diversity is shared among crew members from a fishing that experienced an outbreak of SARS-CoV-2 with a high attack rate. Despite the scenario being highly conducive to transmission, we identified all of the hallmarks of a narrow bottleneck.

### Deep mutational scanning: systematically profiling the impact of viral mutations

Deep mutational scanning (DMS) is an experimental approach used to determine the phenotype of a massive number protein variants simultaneously. In contrast with earlier techniques, DMS is not limited to analyzing a set of mutations selected using *a priori* knowledge about their possible significance. Instead, DMS provides an *unbiased* approach to profile the effects of every possible mutation in a protein sequence. The unbiased nature of DMS allows for the systematic investigation of phenotypes ranging from biochemical properties to evolutionary constraints. In this section, I'll focus on how DMS has been used to study viruses.

#### An overview of the design of deep mutational scans

The design of a mutational scan involves three steps; creating a library of protein variants, imposing a selective pressure, and comparing the library's composition before and after selection. The exact details of each step depends on the context of the experiment. For example, multiple approaches are available to generate the mutant library, including random mutagenesis, codon-directed mutagenesis, and mutant oligonucleotide synthesis. Additionally, there infinite types of selection to apply to the mutant library, each of which dictates the genotype-phenotype relationship. Finally, the method for calculating the impact of each genotype on the measure phenotype depends on the library design, type of selection, and sequencing approach.

Although every element of this experimental design is important, one of the most crucial aspects is the platform in which the deep mutational scan takes place. For instance, reverse genetics systems make it possible to generate libraries of mutant viruses and infect cells under a wide array of conditions. Reverse genetics and cell-based assays have been used to perform DMS on viral accessory, structural, and receptor proteins. However, this approach is not without significant limitations. For example, biosafety is a concern, as the manipulation of 'live' viral genomes poses risks of accidental release or the creation of more virulent strains, necessitating strict containment and safety protocols.

Surface display offers a safe, alternative, approach to study the properties of viral proteins. In this case, DMS is achieved by displaying a library of mutant peptides on the surface of a cell, or virus, while maintaining the link between genotype and phenotype. There are many types of surface display and each has it's advantages and disadvantages. Yeast can display large libraries of mutant peptides, but the maximum size of the displayed peptide is smaller than most viral proteins. Bacteriophage are another popular choice due to superior library size and ability to map mutations on an entire protein. However, the large library size comes at the cost of using small linear peptides that lack the structural context of the folded protein. Finally, mammalian cells can display entire viral proteins while preserving their structure and relevant post-translational modifications. However, as with all surface display methods, mammalian cell display is limited to measuring biophysical properties like binding and stability rather than infection.

Recently, a novel approach has been developed to perform deep mutational scanning on viral glycoproteins using lentiviral pseudotyping. Similar to other cell-based assays, this pseudotyping approach can capture the effect of mutants on *infection*. However, unlike the reverse genetics systems mentioned above, the restricted replication of pseudotyped lentiviruses addresses biosafety concerns, making this system applicable to highly pathogenic viruses that require high levels of bio-containment.

#### Deep mutational scanning has broadened our understanding of viruses

All of these DMS platforms have led to significant breakthroughs in our understanding of viruses. DMS has been used to study the effect of mutations in viral proteins on fitness by identifying mutations that augment or attenuate viral replication, alter affinity to the host receptor, and change the thermo-stability of viral proteins. Additionally, DMS has been used to assess how mutations in viral proteins lead to resistance against host immunity and antivirals. Additionally, using the lentiviral approach described above, along with a biophysical model of antibody binding, DMS has been used to map the antibody binding footprints of polyclonal sera for viruses like HIV and SARS-CoV-2. Finally, DMS is an excellent tool for studying viral evolution. It's been used to identify adaptive mutations that could increase the risk of zoonosis of pandemic influenza virus. By performing DMS in different strains of the same virus, groups have identified non-linear combinations of mutations in HIV, SARS-CoV2, and Influenza.

#### Overcoming hurdles in the analysis of mutational data

Although DMS has proven incredibly useful for studying viruses, there are still some significant limitations. For instance, the number of mutant sequences in the library pales in comparison to the sequence space of available to an evolving protein. Moreover, the effects of mutations often combine in non-linear ways due to epistasis. This poses an issue, because epistasis plays a significant role in viral evolution. For example, it's been shown that epistasis between mutations in SARS-CoV-2 Spike have allowed it to evade immunity while retaining high binding affinity to the host receptor.

Another limitation of DMS stems from the complexity of the analysis. Given the vast number of experimental designs and questions, it's challenging to develop a consistent approach for analyzing mutational scanning data. As a result, the analysis can impose a significant bottleneck for those lacking computational expertise. There have been attempts to develop user friendly softwares to address these hurdles. For example, programs like `dms-tools` and `Enrich` provide consistent statistical frameworks for inferring the impact of mutations from DMS data. Additionally, libraries like `mutagenesis_visualization` are aimed at at simplifying the visualization steps of the analysis.

However, until recently, there wasn't a straightforward approach for analyzing mutation-based data in the context of a 3D protein structure. Most researches were forced to take an `ad hoc` approach involving multiple steps and softwares. To address this, our lab developed `dms-view`, an in interactive tool the integrated key summary visualizations with an interactive 3D protein structure.
